src/main/java/logisticspipes/proxy/buildcraft/BCRenderTESR.java

```diff
@@ -10,32 +12,38 @@
 
 public class BCRenderTESR implements IBCRenderTESR {
 
-    private final Method renderGatesWires;
-    private final Method renderPluggables;
-
-    @SneakyThrows(Exception.class)
-    BCRenderTESR() {
-        renderGatesWires = PipeRendererTESR.class.getDeclaredMethod(
-                "renderGatesWires",
-                new Class[] { TileGenericPipe.class, double.class, double.class, double.class });
-        renderGatesWires.setAccessible(true);
-        renderPluggables = PipeRendererTESR.class.getDeclaredMethod(
-                "renderPluggables",
-                new Class[] { TileGenericPipe.class, double.class, double.class, double.class });
-        renderPluggables.setAccessible(true);
+    private static final MethodHandle renderGatesWires;
+    private static final MethodHandle renderPluggables;
+
+    static {
+        MethodHandles.Lookup lookup = MethodHandles.lookup();
+        renderGatesWires = getHandle(lookup, "renderGatesWires");
+        renderPluggables = getHandle(lookup, "renderPluggables");
+    }
+
+    private static MethodHandle getHandle(MethodHandles.Lookup lookup, String methodName) {
+        try {
+            Method method = PipeRendererTESR.class.getDeclaredMethod(
+                    methodName,
+                    new Class[] { TileGenericPipe.class, double.class, double.class, double.class });
```

Comment:
Ah, I missed this, you don't need the `new Class[] {` or `}` here, it's a vararg function.

Reply:
Even better :)